runtime/runtime/src/lib.rs

```
@@ -1286,7 +1286,7 @@ impl Runtime {
         // We first process local receipts. They contain staking, local contract calls, etc.
         if let Some(prefetcher) = &mut prefetcher {
             prefetcher.clear();
-            let _queue_full = prefetcher.input_receipts(&local_receipts);
+            _ = prefetcher.prefetch_receipts_data(&local_receipts);
```

(dito) Optional: add a comment to explain why error is ignored